.github/workflows/nightly-soak.yml

```
@@ -0,0 +1,506 @@
+          # De-dup: avoid filing a fresh issue every night during a
+          # protracted red streak. Search for an open issue with the
+          # canonical title prefix; if one exists, comment instead.
+          title="nightly-soak: 3 consecutive failures — residual flake regression"
+          existing=$(gh issue list \
+            --repo "${OWNER}/${REPO}" \
+            --search "in:title \"${title}\" state:open" \
+            --json number,title \
+            --jq '.[0].number // empty')
+
+          body=$(cat <<EOF
+\`nightly-soak\` has failed three consecutive scheduled runs. This
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)